CMakeLists.txt

```diff
@@ -45,6 +45,8 @@ set(BUILD_TESTING NO)
 set(COLLECTIONS_SINGLE_MODULE YES)
 set(COLLECTIONS_FOUNDATION_TOOLCHAIN_MODULE YES)
 
+set(SwiftFoundation_MACRO "" CACHE STRING "Path to Foundation macro plugin")
```

Comment:
Thanks for updating to add this - could we update this PR to remove the `FoundationEssentials` --> `FoundationMacros` target dependency and update the `-plugin-path` argument passed to the build?

Reply:
Yeap!